mezzanine/accounts/defaults.py

```diff
@@ -56,6 +56,14 @@
 )
 
 register_setting(
+    name="ACCOUNTS_PROFILE_INLINE_CLASS",
+    description=_("Dotted package path of the profile inline "
```

Comment:
IMHO this could explain a bit better that this appears in the User admin. Something like "Dotted package path and class name of the profile inline admin that will be added to the User admin, when `mezzanine.accounts` is installed and `settings.AUTH_PROFILE_MODULE` is defined."